### PR TITLE
[new release] obus (1.2.1)

### PR DESCRIPTION
**Pull request by Freyr666**

### CODE REVIEW

File: packages/obus/obus.1.2.1/opam [hunk 1]
```diff
@@ -0,0 +1,30 @@
...
+bug-reports: "https://github.com/ocaml-community/obus/issues"
+dev-repo: "git+https://github.com/ocaml-community/obus.git"
+license: "BSD-3-Clause"
+
+build: [
+  [ "dune" "build" "-p" name "-j" jobs ]
+]
+
+depends: [
+  "dune"
```
Comment (mseri) [inline packages/obus/obus.1.2.1/opam]:
```suggestion
  "dune" {>= "1.1"}
```
I am not sure if `lang menhir 2.0` requires a higher lower bound though

Comment (Freyr666) [reply packages/obus/obus.1.2.1/opam]: Do the previous releases need this too?

Comment (kit-ty-kate) [reply packages/obus/obus.1.2.1/opam]: No, previous releases did not use dune.

Comment (kit-ty-kate) [reply packages/obus/obus.1.2.1/opam]:
```suggestion
  "dune" {>= "1.4"}
```
Apparently `(using merhir 2.0)` requires dune 1.4 (see https://github.com/ocaml/dune/pull/863). Dune is not able to check which plugin require which version, see https://github.com/ocaml/dune/issues/2957

Comment (Freyr666) [reply packages/obus/obus.1.2.1/opam]: 1.2.0 released used dune as well, this release contains only trivial fixes for ppxlib and lwt compatibility. I'll add this constraint to both 1.2.0 and 1.2.1 then.

**Comment by mseri** [comment]
Thanks!

**Pull request merged**